commands/cli.go

```diff
@@ -27,12 +28,30 @@ func CreateCli(version string) *cli.App {
 			Name:  "exclude-region",
 			Usage: "regions to exclude",
 		},
+		cli.StringFlag{
+			Name:  "older-than",
+			Usage: "duration of resources' creation date to delete from",
```

Comment:
A bit hard to understand. Perhaps "Only delete resources older than this specified value. Can be any valid Go duration, such as 1d or 8h."